Changelog.md

```diff
@@ -0,0 +1,8 @@
+
+# Changelog
+
+## GHC 9.4.7
+
+* cabal-install is now version 3.10
+* `prune-juice` has been removed. It is [deprecated](https://github.com/dfithian/prune-juice#readme).
+  Use `-Wunused-packages` [instead](https://downloads.haskell.org/~ghc/9.4.7/docs/users_guide/using-warnings.html?highlight=wunused%20packages#ghc-flag--Wunused-packages).
```

Comment:
Is this an advisory for users of this image (I assume so, since I see no reference to `-Wunused-packages` in this diff)? 

If so please make that explicit, e.g.
```suggestion
  Users of this image should use `-Wunused-packages` [instead](https://downloads.haskell.org/~ghc/9.4.7/docs/users_guide/using-warnings.html?highlight=wunused%20packages#ghc-flag--Wunused-packages).
```